sdk/src/types/block/output/feature/staking.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+/// Stakes IOTA coins to become eligible for committee selection, validate the network and receive Mana rewards.
+#[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Hash, packable::Packable)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct StakingFeature {
+    /// The amount of IOTA coins that are locked and staked in the containing account.
```

Comment:
```suggestion
/// Stakes coins to become eligible for committee selection, validate the network and receive Mana rewards.
#[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Hash, packable::Packable)]
#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
pub struct StakingFeature {
    /// The amount of coins that are locked and staked in the containing account.
```
Not sure if we should do that if it's also used in Shimmer

Reply:
Yeah I copy pasted from the TIP